package.json

```diff
@@ -38,7 +38,7 @@
     "lint:grunt:report": "grunt lint --checkstyle",
     "lint": "mkdir -p build2 && run-p -c lint:eslint lint:grunt lint:stylelint",
     "lint:report": "mkdir -p build2 && run-p -c lint:eslint:report lint:grunt:report lint:stylelint:report",
-    "test:dev": "yarn build:dev && karma start",
+    "test:dev": "grunt assets && karma start",
```

Comment:
we don't need to build the app to run tests?

Reply:
no we dont. we use `build:dev` task previously because it needs those json files (`propertiesToJSON`) which is part of `build:dev` and now I moved to `assets` task.